docs/site/Setting-debug-strings.md

```diff
@@ -363,3 +363,31 @@ of the `@loopback/cli` module.
     </tr>
   </tbody>
 </table>
+
+## Adding debugs
+
+As seen before, LoopBack as built-in debug strings to help with debugging.
+
+LoopBack use internally the `debug` package, even if there's no mandate for
+LoopBack users to use the same library, you can use this package in your
+application to help debugging.
+
+To do so, you can define your own debug strings like demonstrated in this
+example:
+
+```ts
+// Import from debug
+import debugFactory from 'debug';
+
+// Define your custom debug string
+const debug = debugFactory('example:debug:factory');
+
+// Use it in your code
+debug('Oops there was an error!');
+```
+
+To debug parts of your app with this custom debug string, you can run the next
+command: `DEBUG=example:debug:factory npm start`
+
+> Read the documentation of [debug](https://www.npmjs.com/package/debug) to know
+> how to customize output messages
```

Comment:
Let's give a more generic pointer, since the `debug` documentation essentially lays out everything about the package itself:

```suggestion
Refer to the [debug](https://www.npmjs.com/package/debug) documentation for more information.
```